### PR TITLE
PyTorch 1.3 Support

### PR DESCRIPTION
**Pull request by guyjacob**
* summary_graph.py:
  * Change ONNX `op.uniqueName()` to `op.debugName()`
  * Removed scope-naming workaround which isn't needed in PyTorch 1.3
* Tests:
  * Naming of trace entries changed in 1.3. Fixed SummaryGraph unit test that checked that
  * Adjusted expected values in full_flow_tests
  * Adjusted tolerance in test_sim_bn_fold
  * Filter some new warnings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -163,7 +163,7 @@ $ source env/bin/activate
 ```
 
 ### Install the package
-If you do not use CUDA 9 in your environment, please refer to [Pytorch website](https://pytorch.org/get-started/locally/) to install the compatible build of Pytorch 1.1 and torchvision 0.3, before installing the package.
+If you do not use CUDA 9 in your environment, please refer to [PyTorch website](https://pytorch.org/get-started/locally/) to install the compatible build of PyTorch 1.3.1 and torchvision 0.4.2, before installing the package.
```
Comment (barrh) [inline README.md]: Pytorch v1.3 default CUDA version is 10.1, not CUDA 9. If you wish to continue testing on cuda9, also requirements.txt will have to be changed to specific URL, vs. "torch".

File: README.md [hunk 2]
```diff
@@ -172,7 +172,7 @@ $ pip3 install -e .
 ```
 This installs Distiller in "development mode", meaning any changes made in the code are reflected in the environment without re-running the install command (so no need to re-install after pulling changes from the Git repository).
 
-PyTorch is included in the ```requirements.txt``` file, and will currently download PyTorch version 1.1.0 for CUDA 9.0.  This is the setup we've used for testing Distiller.
+PyTorch is included in the ```requirements.txt``` file, and will currently download PyTorch version 1.3.1 for CUDA 9.0.  This is the setup we've used for testing Distiller.
```
Comment (barrh) [inline README.md]: See comment above.

File: README.md (hunk 1 above)
Comment (guyjacob) [reply README.md]: Good point, thanks. We actually test don't test on CUDA 9 anymore, we use the default PyTorch package. I'll modify the README.

File: README.md (hunk 2 above)
Comment (guyjacob) [reply README.md]: See response above

**Pull request merged**